BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -355,7 +356,7 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
 
     // Mailto URLs can be encountered if there is a hyperlink for sending an email.
     if ([[[[request URL] scheme] lowercaseString] isEqualToString:@"mailto"]) {
-        [[UIApplication sharedApplication] openURL:[request URL]];
+        [[UIApplication box_sharedApplication] box_openURL:[request URL]];
```

Comment:
@scottliubox is there any resaons behind those implementations (this is the category that was in the app)